src/App.jsx

```diff
@@ -14,70 +16,64 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [SelectedGoods, setSelectedGoods] = useState('Jam');
```

Comment:
no needs to write with a capital letter. And these goods aren't selected, so just use `good`
```suggestion
  const [goods, setGoods] = useState('Jam');
```

Reply:
Im use variable 'good' in map, and if i write goods - lint sent me about error, because 'goods' used upper in array. Capital letter was typo, i'll correct it.